packages/clay-card/src/ClayUserCard.js

```diff
@@ -0,0 +1,183 @@
+import 'clay-link';
+import 'clay-dropdown';
+import 'clay-sticker';
+import './ClayCard';
+import Component from 'metal-component';
+import defineWebComponent from 'metal-web-component';
+import Soy from 'metal-soy';
+import {Config} from 'metal-state';
+
+import actionItemsValidator from './action_items_validator';
+import templates from './ClayUserCard.soy.js';
+
+/**
+ * Metal ClayUserCard component.
+ */
+class ClayUserCard extends Component {}
+
+/**
+ * State definition.
+ * @static
+ * @type {!Object}
+ */
+ClayUserCard.STATE = {
+	/**
+	 * List of items to display in the actions menu.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {?array|undefined}
+	 * @default undefined
+	 */
+	actionItems: actionItemsValidator,
+
+	/**
+	 * Flag to indicate if the card is disabled or not.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {?bool|undefined}
+	 * @default false
+	 */
+	disabled: Config.bool().value(false),
+
+	/**
+	 * CSS classes to be applied to the element.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {?string|undefined}
+	 * @default undefined
+	 */
+	elementClasses: Config.string(),
+
+	/**
+	 * Id to be applied to the element.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {?string|undefined}
+	 * @default undefined
+	 */
+	id: Config.string(),
+
+	/**
+	 * Alternate text of the image.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {?string|undefined}
+	 * @default undefined
+	 */
+	imageAlt: Config.string(),
+
+	/**
+	 * Source of the image to be rendered in the visual area of the card.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {?string|undefined}
+	 * @default undefined
+	 */
+	imageSrc: Config.string(),
+
+	/**
+	 * Initials of the user.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {?string|undefined}
+	 * @default undefined
+	 */
+	initials: Config.string(),
+
+	/**
+	 * Name to be applied to the input element.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {?string|undefined}
+	 * @default undefined
+	 */
+	inputName: Config.string(),
+
+	/**
+	 * Value to be applied to the input element.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {?string|undefined}
+	 * @default undefined
+	 */
+	inputValue: Config.string(),
+
+	/**
+	 * Name of the user.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {!string}
+	 * @default undefined
+	 */
+	name: Config.string().required(),
+
+	/**
+	 * Flag to indicate if the card is selectable or not.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {?bool|undefined}
+	 * @default false
+	 */
+	selectable: Config.bool().value(false),
+
+	/**
+	 * Flag to indicate if the card is selected or not.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {?bool|undefined}
+	 * @default false
+	 */
+	selected: Config.bool().value(false),
+
+	/**
+	 * The path to the SVG spritemap file containing the icons.
+	 * @instance
+	 * @memberof ClayUserCard
+	 * @type {!string}
```

Comment:
Maybe we can leave `?String|undefined` since it is not used **icon** in some cases. What do you think?